src/Forms/FieldsValidator.php

```diff
@@ -2,11 +2,21 @@
 
 namespace SilverStripe\Forms;
 
+use SilverStripe\Dev\Deprecation;
+
 /**
  * Validates the internal state of all fields in the form.
+ *
+ * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it
```

Comment:
```suggestion
 * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it
```
This is where the word "functionality" instead of "API" is gonna bite us.
There is equivalent functionality, it's just not in its own class.

Would it be better to say "Will be replaced with functionality inside Silverstripe\Forms\Form::validate()"? ASsuming that's where the logic will be - doesn't seem to have been done in the CMS 6 PR yet

Reply:
Updated